seed/views/portfoliomanager.py

```diff
@@ -260,16 +252,24 @@ def get_list_of_report_templates(self):
             _log.debug('Found template,\n id=' + str(t['id']) + '\n name=' + str(t['name']))
             if 'hasChildrenRows' in t and t['hasChildrenRows']:
                 _log.debug('Template row has children data request rows, trying to get them now')
-                children_url = \
-                    'https://portfoliomanager.energystar.gov/pm/reports/templateTableChildrenRows/TEMPLATE/{0}'.format(
-                        t['id']
-                    )
+                children_url = f'https://portfoliomanager.energystar.gov/pm/reports/templateChildrenRows/TEMPLATE/{t["id"]}'
+
                 # SSL errors would have been caught earlier in this function and raised, so this should be ok
                 children_response = requests.get(children_url, headers=self.authenticated_headers)
                 if not children_response.status_code == status.HTTP_200_OK:
                     raise PMExcept('Unsuccessful response from child row template lookup; aborting.')
                 try:
-                    child_object = json.loads(children_response.text)
+                    # the data are now in the string of the data key of the returned dictionary with an excessive amount of
+                    # escaped doublequotes.
+                    # e.g., response = {"data": "{"customReportsData":"..."}"}
+                    decoded = json.loads(children_response.text)  # .encode('utf-8').decode('unicode_escape')
```

Comment:
This is non-ideal, but the response is a json with a serialized JSON string. There also includes unnecessary double quotes in front of the array and dict, which need to be removed.